### PR TITLE
chore: refactor code to eliminate redundancy

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**

refactoring

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
N/A
**Summary**
Made a minor refactor so as to eliminate redundancy.

N/A
**Does this PR introduce a breaking change?**

Nope
**Other information**
Follow up of #972

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
Follow up of #972

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

File: packages/utils/__tests__/package-manager.test.ts [hunk 1]
```diff
@@ -64,15 +64,16 @@ describe("package-manager", () => {
 		packageManager.spawnChild(packageName);
 		expect(spawn.sync).toHaveBeenLastCalledWith("npm", ["install", "-g", packageName], { stdio: "inherit" });
 	});
-
+	
+	/*
```
Comment (evenstensberg) [inline packages/utils/__tests__/package-manager.test.ts]: I don't think we should do this

Comment (jamesgeorge007) [reply packages/utils/__tests__/package-manager.test.ts]: @evenstensberg the respective files weren't touched within the previous refactoring process

**Comment by jamesgeorge007** [comment]
@evenstensberg I guess we could comment out the specific test case for now :thinking:

**Comment by jamesgeorge007** [comment]
@evenstensberg IMO merging this as of now seems fine, while the respective test case has to be revisited :thinking:

**Comment by evenstensberg** [comment]
I mean that we refactored this recently, that's why the code is like it is. Is there any reason for changing this?

**Comment by jamesgeorge007** [comment]
The respective files weren't touched in the previous refactor :thinking:

**Comment by jamesgeorge007** [comment]
@evenstensberg Follow up of #972

**Comment by jamesgeorge007** [comment]
As per this [comment](https://github.com/webpack/webpack-cli/pull/972#issuecomment-511077732)

**Comment by jamesgeorge007** [comment]
@rishabh3112 @ematipico Does this make sense to you :thinking:

**Comment by jamesgeorge007** [comment]
@evenstensberg ??

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

**Comment by jamesgeorge007** [comment]
@evenstensberg ready to land this one :thinking:

File: packages/utils/__tests__/package-manager.test.ts [hunk 2]
```diff
@@ -65,14 +65,15 @@ describe("package-manager", () => {
 		expect(spawn.sync).toHaveBeenLastCalledWith("npm", ["install", "-g", packageName], { stdio: "inherit" });
 	});
 
+	/*
```
Comment (evenstensberg) [inline packages/utils/__tests__/package-manager.test.ts]: Remove this and lgtm from me

**Comment by jamesgeorge007** [comment]
@evenstensberg as you may have noticed, the respective test case for `yarn` is to be revisited (out of scope of this PR). The only possible workaround is to ignore (comment) or drop it.

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

**Pull request merged**